internal/pkg/metrics/kyma.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 const (
-	metricKymaState   = "lifecycle_mgr_kyma_state"
-	metricModuleState = "lifecycle_mgr_module_state"
+	MetricKymaState   = "lifecycle_mgr_kyma_state"
```

Comment:
Actually I come across this reference const to test code in several conversations.

To be honest, the [reply](https://stackoverflow.com/a/23374335) from this guy is more or less aligned with my idea.

If we want to verify the const value should not be changed, then we should implement a test to specifically test the naming consistency, in this case, we indicate this purpose more specifically. 

I don't think reference this const value in other test code for other purpose is bad practice, why do we restrict it? Let's say if we need to reuse this value in 10 other different test scenario? How do we deal with it? define 10 different const in each individual test?

And in some complicated test scenario, if we just define another variable without reference the value in production code, it will even bring much more confusion to us, because we lack of the context of this value.

So it really depends on the test, we can't treat it as a general rule for all cases.

Reply:
Another argument from the code iteration point of view, we should not expect those values to never change.

Take these metrics as an example, it may have the request coming to rename it. Then if we need to adapt this change, with the reference approach, just need to rename the constant, adjust the naming consistent unit test, done. But if follow the idea never reference production code, it may end up into full text search and reply, even more complicate case like some value in test code is concatenated dynamically.

Reply:
let's put it in another issue, to make it more awareness https://github.com/kyma-project/lifecycle-manager/issues/1109, also this task is not related to this issue.